src/dlgprefreplaygain.cpp

```diff
@@ -108,8 +108,7 @@ void DlgPrefReplayGain::slotUpdateReplayGainBoost() {
 
 void DlgPrefReplayGain::setLabelCurrentReplayGainBoost(int value) {
     LabelCurrentReplayGainBoost->setText(
-            QString(tr("%1 dB (average %2 dB)")).arg(
-                  QString().sprintf("%+d", value), QString::number(value - 14)));
+            QString(tr("Adjust by %1 dB")).arg(QString().sprintf("%+d", value)));
```

Comment:
My experience is just the other way round. When I have first used Mixxx, I was totally lost what might the use of the boost slider. But it was clear after reading about the -14 dB average somewhere. Without that info, there is no way to find out to which level Mixxx will normalize the tracks. When I got it right, radio DJs need the info, to pick the right setting to be complaint with the station rules.  

We should not mention to add a +6 dB boost to the -14 dB Replaygain. This works for Mixxx 1.11 because it compensates the -6 dB headroom in th gain code. But since it is gone now, a 6 dB boost will result is clipping of almost the half of average tracks. IMHO you should not add a positive boost unless you know what you are doing. You can safely add a negative boost and it is required for some collections, because it turns out that the -14 dB average is too high for some high dynamic tracks.  EBU R128 uses -23 dB 

Below is a random snipped of some of my tracks in the replayGain column of the Mixxx library:
 
"0.262724"
"0.712033"
"0.497737"
"0.33037"
"0.358509"
"0.499459"
"0.316957"
"0.969394"
"0.313328"
"0.307964"
"0.92151"
"0.447198"
"0.496592"
"0.447198"
"0.534564"
"0.280221"
"1.06906"
"0.367282"
"0.265155"